pkg/cmd/container/wait.go

```diff
@@ -49,29 +48,27 @@ func Wait(ctx context.Context, client *containerd.Client, reqs []string, options
 	var allErr error
 	w := options.Stdout
 	for _, container := range containers {
-		if waitErr := waitContainer(ctx, w, container); waitErr != nil {
+		code, waitErr := WaitContainer(ctx, container)
+		fmt.Fprintln(w, code)
+		if waitErr != nil {
 			allErr = multierror.Append(allErr, waitErr)
 		}
 	}
 	return allErr
 }
 
-func waitContainer(ctx context.Context, w io.Writer, container containerd.Container) error {
+func WaitContainer(ctx context.Context, container containerd.Container) (code int64, err error) {
```

Comment:
This should be `pkg/containerutil.Wait()`
https://github.com/containerd/nerdctl/blob/main/pkg/containerutil/containerutil.go

Reply:
If we want to keep the pattern consistent with other recent refactoring, instead of making methods public, we can also move stdout to cmd from pkg. 
https://github.com/containerd/nerdctl/pull/2333
https://github.com/containerd/nerdctl/pull/2342/files